internal/config/config.go

```diff
@@ -600,7 +600,9 @@ func (c *Config) GetAttestationConfig() AttestationCfg {
 		return c.Attestation.AWSNitroTPM
 	}
 	if c.Attestation.AzureSEVSNP != nil {
-		return c.Attestation.AzureSEVSNP
+		cp := *c
+		cp.Attestation.AzureSEVSNP.modifyToMarshalNumericalVersionValues()
+		return cp.Attestation.AzureSEVSNP
```

Comment:
```suggestion
		cp := *c.Attestation.AzureSEVSNP.
		cp.modifyToMarshalNumericalVersionValues()
		return cp
```
nit: no need to make a copy of the whole config